src/main/java/tutoraid/model/ModelManager.java

```diff
@@ -22,8 +24,9 @@
  * Represents the in-memory model of the student book data.
  */
 public class ModelManager implements Model {
-    private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
+    public static List<Student> allStudents = new ArrayList<>();
```

Comment:
This causes stylecheck to fail but I don't know how to resolve it. Lesson needs to access this to initialise the Students object